### ISSUE TITLE
new platform

### ISSUE THREAD

**Issue by REJack** [Templarian/WindowsIcons#213]
Hey Templarian, 

do you have any plans when you move the Modern UI Icons to your new platform?

**Comment by Templarian** [comment]
Not really. I might actually abandon the Modern UI Icons project entirely as the next icon pack will be focused on the new icon style Windows 10 uses.

Not 100% sure what to do with this legacy icon pack at the moment.

**Comment by REJack** [comment]
ok thanks for the fast answer, then i need to wait, do you know if there is any Style Guide for the Win10 Icons?
